From eac2996fd96321afd4f1a6719ed6fa1339568802 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Fri, 7 Mar 2014 14:01:28 +0000
Subject: [PATCH 058/102] java: Split long lines in generated output, and add
 other whitespace.

No functional change.

(cherry picked from commit 8c219636a272c4288f22392ff27b9c04580aecb4)
---
 generator/java.ml | 13 ++++++++++---
 1 file changed, 10 insertions(+), 3 deletions(-)

diff --git a/generator/java.ml b/generator/java.ml
index 73ca3df..746b3cd 100644
--- a/generator/java.ml
+++ b/generator/java.ml
@@ -1151,20 +1151,24 @@ and generate_java_struct_return typ jtyp cols =
 and generate_java_struct_list_return typ jtyp cols =
   pr "  cl = (*env)->FindClass (env, \"com/redhat/et/libguestfs/%s\");\n" jtyp;
   pr "  jr = (*env)->NewObjectArray (env, r->len, cl, NULL);\n";
+  pr "\n";
   pr "  for (i = 0; i < r->len; ++i) {\n";
   pr "    jfl = (*env)->AllocObject (env, cl);\n";
+  pr "\n";
   List.iter (
     function
     | name, FString ->
         pr "    fl = (*env)->GetFieldID (env, cl, \"%s\", \"Ljava/lang/String;\");\n" name;
-        pr "    (*env)->SetObjectField (env, jfl, fl, (*env)->NewStringUTF (env, r->val[i].%s));\n" name;
+        pr "    (*env)->SetObjectField (env, jfl, fl,\n";
+        pr "                            (*env)->NewStringUTF (env, r->val[i].%s));\n" name;
     | name, FUUID ->
         pr "    {\n";
         pr "      char s[33];\n";
         pr "      memcpy (s, r->val[i].%s, 32);\n" name;
         pr "      s[32] = 0;\n";
         pr "      fl = (*env)->GetFieldID (env, cl, \"%s\", \"Ljava/lang/String;\");\n" name;
-        pr "      (*env)->SetObjectField (env, jfl, fl, (*env)->NewStringUTF (env, s));\n";
+        pr "      (*env)->SetObjectField (env, jfl, fl,\n";
+        pr "                              (*env)->NewStringUTF (env, s));\n";
         pr "    }\n";
     | name, FBuffer ->
         pr "    {\n";
@@ -1173,7 +1177,8 @@ and generate_java_struct_list_return typ jtyp cols =
         pr "      memcpy (s, r->val[i].%s, len);\n" name;
         pr "      s[len] = 0;\n";
         pr "      fl = (*env)->GetFieldID (env, cl, \"%s\", \"Ljava/lang/String;\");\n" name;
-        pr "      (*env)->SetObjectField (env, jfl, fl, (*env)->NewStringUTF (env, s));\n";
+        pr "      (*env)->SetObjectField (env, jfl, fl,\n";
+        pr "                              (*env)->NewStringUTF (env, s));\n";
         pr "    }\n";
     | name, (FBytes|FUInt64|FInt64) ->
         pr "    fl = (*env)->GetFieldID (env, cl, \"%s\", \"J\");\n" name;
@@ -1188,8 +1193,10 @@ and generate_java_struct_list_return typ jtyp cols =
         pr "    fl = (*env)->GetFieldID (env, cl, \"%s\", \"C\");\n" name;
         pr "    (*env)->SetLongField (env, jfl, fl, r->val[i].%s);\n" name;
   ) cols;
+  pr "\n";
   pr "    (*env)->SetObjectArrayElement (env, jr, i, jfl);\n";
   pr "  }\n";
+  pr "\n";
   pr "  guestfs_free_%s_list (r);\n" typ;
   pr "  return jr;\n"
 
-- 
1.9.3

